mspa_parsed_consent.go

```diff
@@ -0,0 +1,181 @@
+package iabconsent
+
+// mspaParsedConsent represents data extract from a Multi-State Privacy Agreement (mspa) consent string.
```

Comment:
nit: match case of MspaParsedConsent in comment

Reply:
Good catch, done!